### ISSUE TITLE
Should we include multiple implementation suggestions?

### ISSUE THREAD

**Issue by bmann** [gitcoinco/ERC-1789#36]
Evan Van Ness has suggested a different implementation of block rewards https://www.evanvanness.com/post/183629647376/a-simple-way-to-fund-more-public-goods-in-ethereum

Should this group broadly speaking work towards a block rewards proposal that can have multiple implementation suggestions and then push forward the one that has the most support?

Or should Evan’s proposal be turned into a separate EIP that is “competitive” with 1789?

Even with the latter, this group could have different people working on both but collaborating in different ways.

**Comment by owocki** [comment]
> Should this group broadly speaking work towards a block rewards proposal that can have multiple implementation suggestions and then push forward the one that has the most support?

this is the way i'm leaning. @lrettig ?

**Comment by lrettig** [comment]
I agree, for now let's explore multiple options here. I think it'll become clear over time if there is one "winner" that has the most support, or if we want to split the initiative and develop several proposals. It's also fairly likely that, like the issuance reduction/delay of the difficulty bomb in Constantinople, we may end up with an array of proposals that are very similar except for a few parameters.

I'd also point out that my [EIP-1789 proposal](https://github.com/lrettig/EIPs/blob/8dd2e7b70a9229fa0408a2f88aa9667566105bf3/EIPS/eip-1789.md) should actually be compatible with all of these ideas since the beneficiary is set to null to begin.

**Comment by glauseWilde** [comment]
✋ raises hand to lead a proposal.  Initial draft #41
